CHANGES.txt

```diff
@@ -15,6 +15,10 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+- ``region`` setting for ``CREATE REPOSITORY`` has been removed. The region is
```

Comment:
"the ``region``"

Reply:
on the next sentence I suppose you can say "it is automatically guessed ... " to avoid repeating "the region)